charts/substra-backend/README.md

```diff
@@ -286,6 +286,25 @@ Else, you must strike a balance: longer durations are more convenient, but risk
 | `oidc.users.channel`                    | The channel to assign OIDC users to (mandatory)                                                                                                                 | `nil`   |
 | `oidc.users.appendDomain`               | As usernames are assigned based on e-mail address, whether to suffix user names with the email domain (john.doe@example.com would then be `john-doe-example`)   | `false` |
 
+### Database connection settings
+
+| Name                                  | Description                                                                                                 | Value      |
+| ------------------------------------- | ----------------------------------------------------------------------------------------------------------- | ---------- |
+| `database.auth.database`              | what DB to connect to                                                                                       | `substra`  |
+| `database.auth.username`              | what user to connect as                                                                                     | `postgres` |
+| `database.auth.password`              | what password to use for connecting                                                                         | `postgres` |
+| `database.auth.credentialsSecretName` | An alternative to giving username and password; must have `DATABASE_USERNAME` and `DATABASE_PASSWORD` keys. | `nil`      |
+| `database.host`                       | Hostname of the database to connect to (defaults to local)                                                  | `nil`      |
+| `database.port`                       | Port of an external database to connect to                                                                  | `5432`     |
+
+### Integrated PostgreSQL settings
```

Comment:
Do we want to keep the "Integrated" denomination here? 
Pros: it's more explicit
Cons: it's not a denomination used else where 🤷

Reply:
I fully agree, maybe I can just call it "PostgreSQL settings" and add a little blurb in the section description to explain what it is.

Reply:
yes, very good idea